### ISSUE TITLE
Feature: limit page pollution

### ISSUE THREAD

**Issue by raswonders** [raswonders/bootstrap-grid-overlay#15]
There are two main sources of DOM pollution which happen automatically after user presses action button. Both of these are removed when page is refreshed.

1. introduction of grid-overlay div element into page's body
2. introduction of scroll and resize listeners

It's irritating especially because it happens even for false positive pages and it might introduce some sort of unexpected performance hindrance to user's experience.

Steps to do:

- [x] postpone adding grid-overlay element to DOM only after first overlay is added
- [x] remove grid-overlay after last overlay was removed
- [ ] postpone adding eventListeners only after first overlay is added
- [ ] remove eventListeners after last overlay was removed

**Comment by raswonders** [comment]
There are two event listeners and their handlers doing almost no work when there is no overlay element being displayed.  Possible reimplementation would require adding/removing these listeners every time grid-overlay element would be added/removed.  That would require saving event handler functions for later removal. Even thought it's probably doable it seems too difficult for tiny gain we get from it. Hence I'm closing it for now.
